libavpipe/src/avpipe_xc.c

```diff
@@ -676,14 +683,12 @@ set_encoder_options(
                 params->seg_duration, seg_duration_ts, params->url);
             av_opt_set(encoder_context->format_context->priv_data, "reset_timestamps", "on", 0);
         }
-        // If I set faststart in the flags then ffmpeg generates some zero size files, which I need to dig into it more (RM).
-        // av_opt_set(encoder_context->format_context->priv_data, "segment_format_options", "movflags=faststart", 0);
-        // So lets use flag_every_frame option instead.
```

Comment:
I'd still keep some info somewhere on why we don't use `faststart`

Reply:
I added a note about faststart just so we have it.  It's not related to fragmented mp4 and DASH/HLS at all - its' for full file playout.